internal/system/agent/executor/compose.go

```diff
@@ -20,19 +31,19 @@ func WasDockerContainerComposeStarted(service string) bool {
 	cmdName := "docker"
 	cmdArgs := []string{"ps"}
 	if cmdOut, err = exec.Command(cmdName, cmdArgs...).Output(); err != nil {
-		fmt.Fprintln(os.Stderr, "There was an error running the docker-compose command: ", err.Error())
+		logs.LoggingClient.Error("error running the docker-compose command", "error message", err.Error())
```

Comment:
Labels will be removed from the logging system (issue #904), so please don't use it.

Reply:
This looks right to me in that you are aligned with the rework of logging output using GoKit as shown in the Core WG call 1-Nov-18 👍